frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/RelationshipEdge/RelationshipEdge.tsx

```diff
@@ -29,7 +36,17 @@ export const RelationshipEdge: FC<Props> = ({
 
   return (
     <>
-      <BaseEdge id={id} path={edgePath} />
+      <BaseEdge
+        id={id}
+        path={edgePath}
+        style={{
+          stroke: data?.isHovered
+            ? 'var(--node-layout)'
+            : 'var(--global-border)',
+          strokeDasharray: '2',
+          transition: 'stroke 0.3s var(--default-timing-function)',
+        }}
```

Comment:
Styles should be implemented in the CSS Module as far as possible.
Perhaps it could be written as follows:

```suggestion
        className={clsx(styles.edge, data?.isHovered && styles.hovered)}
```

Reply:
That's right 👍
6cc25f891f6053f70824cd05207ad857364da34f